addons/l10n_sa/__manifest__.py

```diff
@@ -2,18 +2,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Saudi Arabia - Accounting',
-    'version': '2.0',
-    'author': 'DVIT.ME',
+    'version': '2.1',
```

Comment:
You don't need to update the version, do you?

Reply:
I don't think so, now that I understand more the upgrade process. When would it be considered as a need (for future cases)?

Reply:
Almost never.
When you need to write a migration script inside of this module, and not in the `upgrade` repo; so in some very specific cases.